File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    var even = 0;
    var odd = 0;
    
    for(var n of num_list){
        if(n%2 == 0){
            even++;
        }else{
            odd++;
        }
    }
    
    answer[0] = even;
    answer[1] = odd;
    
    return answer;
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.kt
class Solution {
    fun solution(num_list: IntArray): IntArray {
        var odd: Int = 0
        var even: Int = 0
        
        for (i in 0..num_list.size - 1) {
            val num = num_list[i]
            if (num % 2 == 0) {
                even++
            } else {
                odd++
            }
        }
        
        return intArrayOf(even, odd)
    }
}